TargetDown alert isn't specific enough when job includes multiple services

**What is missing?**
When a job encompasses multiple services, the [target down message](https://github.com/coreos/kube-prometheus/blob/master/jsonnet/kube-prometheus/alerts/general.libsonnet#L10) isn't specific enough to identify which service has down targets.

**Why do we need it?**
Allowing jobs with multiple targets / services seems reasonable. The alert already groups `BY (job, namespace, service)`.

**Environment**
* kube-prometheus version:  master

**Anything else we need to know?**:
I would propose that the message be changed to read `the {{ $labels.job }}/{{ $labels.service }} targets`. I am more than happy to create that PR if that seems reasonable.

Hmm. I'm trying to `make generate-in-docker` and running into errors both on master and my fork.

```
>> Compiling assets and generating Kubernetes manifests
docker run --rm -e http_proxy -e https_proxy -e no_proxy -u="501:20" -v "/Users/jason/Library/Caches/go-build:/.cache/go-build" -v "/Users/jason/Development/forks/kube-prometheus:/go/src/github.com/coreos/kube-prometheus:Z" -w "/go/src/github.com/coreos/kube-prometheus" quay.io/coreos/jsonnet-ci make  generate
rm -rf vendor
jb install
rm -rf manifests
./build.sh examples/kustomize.jsonnet
+ set -o pipefail
+ rm -rf manifests
+ mkdir -p manifests/setup
+ jsonnet -J vendor -m manifests examples/kustomize.jsonnet
+ xargs '-I{}' sh -c 'cat {} | gojsontoyaml > {}.yaml; rm -f {}' -- '{}'
RUNTIME ERROR: couldn't open import "kube-prometheus/kube-prometheus.libsonnet": no match locally or in the Jsonnet library paths
	examples/kustomize.jsonnet:2:4-54	thunk <kp> from <$>
	examples/kustomize.jsonnet:11:88-90
	<std>:1268:24-25	thunk from <function <anonymous>>
	<std>:1268:5-33	function <anonymous>
	examples/kustomize.jsonnet:11:71-106	thunk <manifests> from <$>


	examples/kustomize.jsonnet:32:1-10	$
	During evaluation

make: *** [Makefile:39: manifests] Error 1
make: *** [generate-in-docker] Error 2
```

i tried poking around in the container but I don't have enough familiarity to say what anything should be doing. Would love to get this thing across the finish line if someone can point me in the right direction.

--- a/jsonnet/kube-prometheus/alerts/general.libsonnet
+++ b/jsonnet/kube-prometheus/alerts/general.libsonnet
@@ -7,7 +7,7 @@
           {
             alert: 'TargetDown',
             annotations: {
-              message: '{{ printf "%.4g" $value }}% of the {{ $labels.job }} targets in {{ $labels.namespace }} namespace are down.',
+              message: '{{ printf "%.4g" $value }}% of the {{ $labels.job }}/{{ $labels.service }} targets in {{ $labels.namespace }} namespace are down.',
             },
             expr: '100 * (count(up == 0) BY (job, namespace, service) / count(up) BY (job, namespace, service)) > 10',
             'for': '10m',

--- a/manifests/prometheus-rules.yaml
+++ b/manifests/prometheus-rules.yaml
@@ -1329,8 +1329,8 @@ spec:
     rules:
     - alert: TargetDown
       annotations:
-        message: '{{ printf "%.4g" $value }}% of the {{ $labels.job }} targets in
-          {{ $labels.namespace }} namespace are down.'
+        message: '{{ printf "%.4g" $value }}% of the {{ $labels.job }}/{{ $labels.service
+          }} targets in {{ $labels.namespace }} namespace are down.'
       expr: 100 * (count(up == 0) BY (job, namespace, service) / count(up) BY (job,
         namespace, service)) > 10
       for: 10m

